modules/net-lb-int/README.md

```diff
@@ -119,12 +119,16 @@ module "ilb" {
   project_id    = var.project_id
   region        = "europe-west1"
   name          = "ilb-test"
-  protocol      = "L3_DEFAULT"
   service_label = "ilb-test"
   vpc_config = {
     network    = var.vpc.self_link
     subnetwork = var.subnet.self_link
   }
+  forwarding_rules_config = {
```

Comment:
we should use the default in the variable default in examples, and show a single example with multiple rules defined

Reply:
in this example, like 127-131 would be removed (we can default the protocol in the variable type too)

Reply:
we do set the default for the variable to "TCP". This is a specific example where we show how to set `L3_DEFAULT`. That's why I specified the protocol.
I'll add one more example with multiple protocols.

Reply:
sorry I closed this by mistake... let me know what you think ;)